extensions/JeremyGamer13/tween.js

```diff
@@ -81,224 +128,295 @@
             const tweened = this[easeMethod](progress, easeDirection);
             return this.multiplierToNormalNumber(tweened, start, end);
         }
+        tweenPlayAnimationPosition(args, util) {
+            const easeMethod = Cast.toString(args.MODE);
+            const easeDirection = Cast.toString(args.DIRECTION);
 
-        // easing functions (placed below blocks for organization)
-        linear(x) {
-            // lol
-            return x;
+            const startx = Cast.toNumber(util.target.x);
+            const starty = Cast.toNumber(util.target.y);
+            const endx = Cast.toNumber(args.ENDX);
+            const endy = Cast.toNumber(args.ENDY);
+            const speed = Cast.toNumber(args.SPEED);
+
+            if (typeof util.stackFrame.loopCounter === 'undefined') {
+                util.stackFrame.loopCounter = 100;
+            }
+            const progress = Cast.toNumber(Math.round(this.clamp(util.stackFrame.loopCounter, 0, 100))) / 100;
+            const tweened = this[easeMethod](progress, easeDirection);
+            const x = Cast.toNumber(this.multiplierToNormalNumber(tweened, endx, startx));
+            const y = Cast.toNumber(this.multiplierToNormalNumber(tweened, endx, startx));

```

Comment:
References to `endx` and `startx` on the right-hand side of `const y = ...` should be updated to `endy` and `starty`. Otherwise, the y position will be the same as the x position.

Reply:
> References to `endx` and `startx` on the right-hand side of `const y = ...` should be changed to `endy` and `starty`. Otherwise, the y position will be the same as the x position. There are 2 occurrences of this issue.

Ah. Not sure how I didn’t catch that, I tested all of my blocks and they work fine.
As for the workflows, I’ll try to reverse that. I originally tried to add a action myself, and later realized that I didn’t want or need to use GitHub.io, just a local server, not knowing there was already workflows inside.